src/universe.cpp

```diff
@@ -25,11 +25,15 @@ void Universe::draw_to_screen() {
     graphics_handler_->update_screen();
 }
 
-void remove_deleted_objects() {
+void Universe::remove_deleted_objects() {
     //TODO
 }
 
+void Universe::add_events_handler(Events *events) {
+    all_game_objects.push_back(events);
+}
+
```

Comment:
This isn't a part of the PR but should defs be here, how did this thing work before having this function? D: